lutris/util/linux.py

```diff
@@ -390,5 +394,66 @@ def gather_system_info():
     system_info["drives"] = LINUX_SYSTEM.get_drives()
     system_info["ram"] = LINUX_SYSTEM.get_ram_info()
     system_info["dist"] = LINUX_SYSTEM.get_dist_info()
+    system_info["arch"] = LINUX_SYSTEM.get_arch()
+    system_info["kernel"] = LINUX_SYSTEM.get_kernel_version()
     system_info["glxinfo"] = glxinfo.GlxInfo().as_dict()
     return system_info
+
+def gather_system_info_str():
+    """Get all relevant system information already formatted as a string"""
+    system_info = gather_system_info()
+    print("Gpus:")
+    print(system_info["gpus"])
+    print("GLX:")
+    print(system_info["glxinfo"])
+    system_info_readable = {}
+    #Add system information
+    system_dict = {}
+    system_dict["OS"] = ' '.join(system_info["dist"])
+    system_dict["Arch"] = system_info["arch"]
+    system_dict["Kernel"] = system_info["kernel"]
+    system_dict["Desktop"] = system_info["env"]["XDG_CURRENT_DESKTOP"]
```

Comment:
You may want to make this line and the one below it call .get() instead of just doing the subscript lookup.  If somebody's not running a full DE and their WM doesn't present that variable to the env, it'll raise an exception.

Reply:
So something like
```suggestion
    system_dict["Desktop"] = system_info["env"].get("XDG_CURRENT_DESKTOP", "No DE found")
```

Reply:
Seems reasonable to me.  Not a maintainer, so I don't want to make a decision on the verbiage, but functionally that's what I was proposing.

Reply:
Good, I'll wait on more info from the maintainers. Maybe they want me to expose more information or remove some lines. Thanks for your comment!